.github/dependabot.yml

```
@@ -20,4 +20,12 @@ updates:
       - "automerge"
     # Set reviewers for pull requests
     reviewers:
+
+  - package-ecosystem: "gradle-wrapper"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    labels:
+      - "dependencies"
+      - "automerge"
       - "AntonButov"
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)